src/Symfony/Component/BrowserKit/Cookie.php

```diff
@@ -92,12 +92,11 @@ public function __toString()
         if (null !== $this->expires) {
             $dateTime = \DateTime::createFromFormat('U', $this->expires, new \DateTimeZone('GMT'));
 
-            if ($dateTime === false) {
-                // this throw will provoke PHP fatal
-                throw new \UnexpectedValueException(sprintf('The cookie expiration time "%s" is not valid.'), $this->expires);
```

Comment:
@kalessil Can you make the change? Moving the validation into the constructor and throw the exception there?